Form-Controls/index.html

```diff
@@ -13,15 +13,57 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
+  <form class="form" action="reesults.html" method="GET">
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
-  </form>
+     <div>
+        <label for="name">Name</label>
+        <input type="text" id="name" required>
+     </div>
+     <div>
+        <label for="email">Email</label>
+        <input type="email" id="email" required>
+     </div>
+     
+     <div>
+      <label for="date">Delivery Date</label>
+      <input type="date" id="date" min="2024-11-10">
+     </div>
+     Colour
+     <div>
+      <label for="red">Red</label>
+      <input type="radio" name="Colour" id="red">
+     </div>
+     <div>
+      <label for="green">Green</label>
+      <input type="radio" name="Colour" id="green">
+     </div>
+     <div>
+      <label for="blue">Blue</label>
+      <input type="radio" name="Colour" id="blue">
+     </div>
+     
+     <div>
+      <label for="size">Size</label>
```

Comment:
You did great Job in this label. But I have a question, If I press Submit button by mistake without choosing my size, form will submit with  the first Size (XS), How we can solve this problem?